lib/astarte_rpc/protocol/proto/vmq_plugin/delete.proto

```diff
@@ -0,0 +1,23 @@
+//
+// This file is part of Astarte.
+//
+// Copyright 2017 Ispirata Srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+message Delete {
+    optional string client_id = 1;
```

Comment:
It would be better to send `device_id` and `realm` separately here. I think having `client_id` in the disconnect call was an error in the first place (because you're leaking the logic of how a client id is built on the broker in DUP, that has to rebuild the client id with the same logic), even more so here where on the VMQ side you're actually re-splitting the `client_id` to derive realm and device id.